cmd/gardenlet/app/app.go

```diff
@@ -451,6 +465,139 @@ func (g *garden) Start(ctx context.Context) error {
 	return nil
 }
 
+// TODO(aaronfern): Remove this code after v1.91 has been released.
+func (g *garden) createNewDWDResources(ctx context.Context, seedClient client.Client) error {
+	// Fetch all namespaces
```

Comment:
```suggestion
	// Fetch all shoot namespaces
```

Or remove the comment, the code is well-readable and self-explanatory.